rootfs/usr/share/opengamepadui/scripts/powertools

```diff
@@ -8,65 +8,79 @@ fi
 
 ## CPU Generics
 cpuBoost(){
-  /usr/bin/echo $1 > /sys/devices/system/cpu/cpufreq/boost
-}
-
-cpuPower(){
-  /usr/bin/cpupower $1 $2 $3
+ state=$1
+  /usr/bin/echo $state > /sys/devices/system/cpu/cpufreq/boost
 }
 
 cpuToggle(){
- /usr/bin/echo $2 > /sys/bus/cpu/devices/cpu${1}/online
+ num=$1
+ state=$2
+ /usr/bin/echo $state > /sys/bus/cpu/devices/cpu${num}/online
 }
 
 smtToggle(){
-  /usr/bin/echo $1 > /sys/devices/system/cpu/smt/control
+ state=$1
+  /usr/bin/echo $state > /sys/devices/system/cpu/smt/control
 }
 
 ## AMD APU
 amdGpuClock(){
-  /usr/bin/echo "s 0 ${1}" > /sys/class/drm/card0/device/pp_od_clk_voltage
-  /usr/bin/echo "s 1 ${2}" > /sys/class/drm/card0/device/pp_od_clk_voltage
-  /usr/bin/echo "c" > /sys/class/drm/card0/device/pp_od_clk_voltage
+  min_clk=$1
+  max_clk=$2
+  card=$3
+  /usr/bin/echo "s 0 ${min_clk}" > /sys/class/drm/${card}/device/pp_od_clk_voltage
+  /usr/bin/echo "s 1 ${max_clk}" > /sys/class/drm/${card}/device/pp_od_clk_voltage
+  /usr/bin/echo "c" > /sys/class/drm/${card}/device/pp_od_clk_voltage
```

Comment:
This change would break any users that are using the update zip, but on an older AUR/systemd-ext version. Can we detect and set a default value for `card`?

E.g.
```bash
if [[ "$3" != "" ]]; then
    card=$3
else
    if [ -L /sys/class/drm/card0 ]; then
        card="card0"
    else
        card="card1"
    fi
fi
```

Reply:
No it won't, the card argument will just get ignored in that case. Adding logic to a file to handle a case that only exists because you can't edit that file makes no sense.

Reply:
You're right. I had this backwards. I was thinking of a case where this new script would be in place but with an older version of OpenGamepadUI. We shouldn't ever run into this problem.